bin/hsw-cli

```diff
@@ -168,7 +168,6 @@ class CLI {
   }
 
   async createAddress() {
-    const account = this.config.str([0, 'account']);
```

Comment:
Why did you remove this?

```
--> hsw-cli address
ReferenceError: account is not defined
```

Reply:
i've pushed the commit with my commented lines by accident, then amended it  in a hurry, and removed not only my lines, but also this.